python-test/features/steps/agentsProvider.py

```diff
@@ -0,0 +1,54 @@
+from behave import given, when, then
+import time
+import functions
+import os
+import docker
+
+email = os.getenv('EMAIL')
+password = os.getenv('PASSWORD')
+
+agentname = os.getenv('AGENT_NAME')
+agentTagKey = os.getenv('TAG_KEY', 'test')
+agentTagValeu = os.getenv('TAG_VALUE', 'true')
+
+
+@given('A valid authentication')
+def get_auth(context):
+    context.token = functions.generate_token(email, password)
+    assert len(context.token) > 0
+
+@when('Create an agent')
+def create_agent(context):
+    context.status_agent, context.agent_data = functions.new_agent(context.token, agentname, agentTagKey, agentTagValeu)
+    assert context.status_agent == 201
+    context.agent_key = context.agent_data['key']
+    context.agent_id = context.agent_data['id']
+    context.agent_channel_id = context.agent_data['channel_id']
```

Comment:
This test is not idempotent. Running it twice fails and we are required to manually delete the agent on Orb UI.
How about deleting the agent at the end of the test?